### PR TITLE
Upgrade to aspnet 5 beta 6 version

### PR DESCRIPTION
**Pull request by Sorting**
Changes to use the latest beta 6 version of aspnet 5.

### CODE REVIEW

**Comment by Sorting** [comment]
FYI - I've just signed the Contributor License Agreement.

File: src/React.Sample.Mvc6/Startup.cs [hunk 1]
```diff
@@ -53,7 +53,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 			if (string.Equals(env.EnvironmentName, "Development", StringComparison.OrdinalIgnoreCase))
```
Comment (Daniel15) [inline src/React.Sample.Mvc6/Startup.cs]: This can be `if (env.IsDevelopment())` now

File: src/React.Sample.Mvc6/React.Sample.Mvc6.xproj [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+    <SchemaVersion>2.0</SchemaVersion>
+    <DevelopmentServerPort>3044</DevelopmentServerPort>
+  </PropertyGroup>
+  <ProjectExtensions>
+    <VisualStudio>
+      <UserProperties />
+    </VisualStudio>
+  </ProjectExtensions>
+  <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.Props" />
+  <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.targets" />
```
Comment (Daniel15) [inline src/React.Sample.Mvc6/React.Sample.Mvc6.xproj]: What happened here? Why is the whole file marked as changed?

Comment (Sorting) [inline src/React.Sample.Mvc6/React.Sample.Mvc6.xproj]: That is a good question. I don't know why React.Sample.Mvc6.xproj was changed. I've corrected the change and pushed a new commit. Thank you for reviewing.

**Comment by Daniel15** [comment]
Thank you! :)

**Comment by Daniel15** [comment]
I'll push this to NuGet tonight.

**Pull request merged**